specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/componentWorkItemConfigs_API.json

```diff
@@ -173,15 +173,91 @@
             "$ref": "./examples/WorkItemConfigDelete.json"
           }
         }
+      },
+      "get": {
+        "description": "Gets specified work item configuration for an Application Insights component.",
+        "operationId": "WorkItemConfigurations_GetItem",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkItemConfigIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/WorkItemConfiguration"
+              }
+            }
+          },
+          "x-ms-examples": {
+            "WorkItemConfigurationsGetDefault": {
+              "$ref": "./examples/WorkItemConfigGet.json"
+            }
+          }
+      },
+      "patch" : {
+        "description": "Update a work item configuration for an Application Insights component.",
+        "operationId": "WorkItemConfigurations_UpdateItem",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkItemConfigIdParameter"
+          },
+          {
+            "name": "WorkItemConfigurationProperties",
+            "description": "Properties that need to be specified to update a work item configuration for this Application Insights component.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/WorkItemCreateConfiguration"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation indicating the config updated by this PATCH call.",
+            "schema": {
+              "$ref": "#/definitions/WorkItemConfiguration"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkItemConfigurationsCreate": {
+            "$ref": "./examples/WorkItemConfigUpdate.json"
```

Comment:
There is some inconsistency in naming of example files - in particular plural (...Config**s**...) for "Create" and "Get". Would be nice to make them consistent

```
./examples/WorkItemConfigDefaultGet.json	
./examples/WorkItemConfigDelete.json
./examples/WorkItemConfigsCreate.json
./examples/WorkItemConfigsGet.json
```

Reply:
There are now two net new operartions that get a specific (single) config. I agree the one S difference is confusing, but wanted to be consistent with the naming. Any suggestions to identify this as an operation on a single resource rather than multiple resources?